be/src/exec/vectorized/hdfs_scanner_orc.cpp

```diff
@@ -457,10 +457,14 @@ Status HdfsOrcScanner::do_get_next(RuntimeState* runtime_state, ChunkPtr* chunk)
     // in the chunk according to the order it's stored. This will lead two chunk from different file to
     // different column order, so we need to adjust the column order according to the input chunk to make
     // sure every chunk has same column order
-    auto convert_to_output = [chunk, &ck]() {
+    auto output_result_chunk = [chunk, &ck]() {
```

Comment:
name `chunk` and `ck` is too confusing. Could you give them more distinguished name?

Reply:
done